lib/tests/streamlit/runtime/scriptrunner/magic_test.py

```diff
@@ -175,11 +175,20 @@ async def myfunc(a):
 """
         self._testCode(CODE_ASYNC_FOR, 1)
 
-    def test_docstring_is_ignored(self):
+    def test_docstring_is_ignored_func(self):
         """Test that docstrings don't print in the app"""
         CODE = """
 def myfunc(a):
     '''This is the docstring'''
     return 42
+"""
+        self._testCode(CODE, 0)
+
+    def test_docstring_is_ignored_async_func(self):
+        """Test that docstrings don't print in the app"""
```

Comment:
nit: would be good to clarify that this is for async functions in this test docstring to differentiate it from the test above